_doc/Interceptors.md

```diff
@@ -8,87 +8,97 @@ next: Error-handlers
 The `@Interceptor` annotation is used to define an interceptor:
```

Comment:
@kaendfinger @cgarciae Can you guys confirm that the snippets on this page are correct according to the new 0.6 API? Thanks